### ISSUE TITLE
Completely freezes when connecting

### ISSUE THREAD

**Issue by superboo07** [Fyustorm/mInetiface#23]
Trying to connect to a valid address entirely freezes the game for me. Reading the log shows absolutely no reason for this. Maybe the issue could be how the server is running on my desktop, while I'm playing on steam deck with the server ip being set to the local ip of my desktop. I'm on the same network, and checking in the intiface dashboard shows its connected, but the game is just frozen no audio goes through and my fans stop blaring. I'm on the linux version of 1.21 for reference.

**Comment by Fyustorm** [comment]
- Which version of the mod do you use ?
- Which game version do you use ?
- Fabric or forge ?
- "Trying to connect to a valid address"  address of what ? Intoface, Minecraft server ?
- Did you install the mod on your client or on the server ?

**Comment by superboo07** [comment]
I am using version 2.1.0, for minecraft 1.21. I am running using fabric, and the valid address is my intiface server (I confirmed the address was valid before filing the bug report.). I installed it on the client as I am playing in a single player world.

**Comment by Fyustorm** [comment]
1.21 ? Not 1.21.x ?
Did you check the "Listen on all network interfaces" on Intiface Central ?
Can you ping your local desktop ip from your deck ?

**Comment by superboo07** [comment]
Yes I can ping it, I checked that before filing the bug report. Also the mod crashed minecraft on the latest 1.21.x so because I didn't care about the bug fixes mojang added I just went down to 1.21.

**Comment by Cyber-Kitty** [comment]
Hello, I've taken over development of mInetiface. Are you still facing this issue? To me it sounds like you might need a newer version of intiface central. Or might be useing the wrong MC version.
